src/components/UtilizationBar/UtilizationBar.js

```diff
@@ -0,0 +1,64 @@
+import { ProgressBar } from '../ProgressBar';
+import PropTypes from 'prop-types';
+import { Tooltip } from '../Tooltip';
+import { OverlayTrigger } from '../OverlayTrigger';
+import React from 'react';
+//import PropTypes from 'prop-types';
+
+const UtilizationBar = ({...props}) => {
+    const availableTooltip = () => {
+      return (
+          <Tooltip id="availableTooltip">
+              Available {props.max - props.now} %
```

Comment:
This should be override-able. Maybe add availableTooltipText and usedTooltipText props?